### ISSUE TITLE
Reproduce K400 with Video-FocalNet-T

### ISSUE THREAD

**Issue by innat** [TalalWasim/Video-FocalNets#2]
Hello. Thank you for sharing this great work. I was trying to reproduce the evaluation results on **K400 test set** using **Video-FocalNet-T**, where it is mentioned in the readme that the top1-acc is **79.8%** while I'm getting around **65%**.

```python
model: video-focalnet-t
clip: 4
frame_rate: 4
input_frame: 8
input_size: 224
```

Here is the [code](https://www.kaggle.com/code/ipythonx/try-kintetics-test-full-set-torch/notebook) we have tried to evaluate v-focalnet on K400-test set (19796 samples). Also, please note, in this code, no crop is used though, but I have also tried `crop=3` but unfortunately no luck. However, the gap between the reported score and computed score is too high.

**Comment by TalalWasim** [comment]
Hi,

Thank you for reaching out.

Please note that we report the results on the Kinetics-400 Validation set, following the common practice of other popular works in literature, where they train on the training set and report on the validation set.

We also provide the log file below to show the results. Additionally, if required, we can make available our resized version of the Kinetics-400 train and validation sets. Please let us know if you would like us to do that.

[log_rank0.txt](https://github.com/TalalWasim/Video-FocalNets/files/12713907/log_rank0.txt).

Kind regards,

**Comment by innat** [comment]
@TalalWasim 
Thanks for your detail response. 

I used K400 test set, [link](https://www.kaggle.com/datasets/ipythonx/k4testset). It's collected from [open-datalab](https://opendatalab.com/). Hopefully it's much as same as yours.

Thanks for sharing the log file, it's helpful. The mian ponts probably are as follows for inference:

```python
INPUT_SIZE: 224
NUM_FRAMES: 8
TEST:
  CROP: true
  NUM_CLIP: 4
  NUM_CROP: 3
```
For now, I didn't use multiple crop, and use only single spatial view. I think not using will not cause major score gap.


In the paper, [page 7](https://arxiv.org/pdf/2307.06947.pdf), it is mentioned as below:

> During inference, we report results as an average across `Nclip × Ncrops`
where a total of `Nclip` clips are uniformly sampled from the video, and for each video, `Ncrops` spatial crops are taken during inference. 

To follow up, I also used `n-clip=3` and iterate inference on multiple clip and average next for a single instance. By doing so, scores improves 65% to 71%.  Here is the [code](https://www.kaggle.com/code/ipythonx/try-kintetics-test-full-set-torch/notebook) I use. Could you please take a quick look and let me know if you notice any invalid setup that might cause such score gap? Note, I use this same setup for other model like uniformer, video-mae etc and it gives reasonable score. Thank you.
